client/state/stored-cards/assembler.js

```diff
@@ -0,0 +1,11 @@
+export const createStoredCard = card => {
```

Comment:
[Here](https://github.com/Automattic/wp-calypso/pull/5251#discussion-diff-62325558) was some discussion. I didn't replace the purchases assembler with a selector yet because it's big and I've been trying to keep the PR size small.